class.jetpack-connection-banner.php

```diff
@@ -22,50 +22,6 @@ static function init() {
 	 */
 	private function __construct() {
 		add_action( 'current_screen', array( $this, 'maybe_initialize_hooks' ) );
-		add_action( 'updating_jetpack_version', array( $this, 'cleanup_on_upgrade' ), 10, 2 );
-	}
-
-	function cleanup_on_upgrade( $new_version = null, $old_version = null ) {
-		if ( version_compare( $old_version, '4.4', '>=' ) && version_compare( $old_version, '5.3', '<' ) ) {
-			delete_option( 'jetpack_connection_banner_ab' );
-		}
```

Comment:
Would there be any cases where a user might have this option saved somewhere. We might want to add it to the list of options to be deleted when the user deletes jetpack.

Reply:
Option is deleted, checked 

https://github.com/Automattic/jetpack/blob/master/class.jetpack-options.php#L535

and

https://github.com/Automattic/jetpack/blob/master/class.jetpack-options.php#L576

Thanks for bringing this up.